packages/@ember/-internals/container/tests/container_test.js

```diff
@@ -736,75 +736,6 @@ moduleFor(
       assert.deepEqual(Object.keys(instance), []);
     }
 
-    '@test instantiating via container.lookup during destruction emits a deprecation'(assert) {
-      let registry = new Registry();
-      let container = registry.container();
-      class Service extends factory() {
-        destroy() {
-          expectDeprecation(() => {
-            container.lookup('service:other');
-          }, /Instantiating a new instance of service:other while the owner is being destroyed is deprecated/);
-        }
-      }
-      registry.register('service:foo', Service);
-      registry.register('service:other', factory());
-      let instance = container.lookup('service:foo');
-      assert.ok(instance, 'precond lookup successful');
-
-      runTask(() => {
-        container.destroy();
-        container.finalizeDestroy();
-      });
-    }
-
-    '@test instantiating via container.lookup during destruction enqueues destruction'(assert) {
```

Comment:
This test should not have been removed. It is testing that things looked up during destruction are properly destroyed (aka the “correctness” fix mentioned in the PR description).